src/test/java/uk/gov/companieshouse/api/accounts/controller/FilingControllerTest.java

```diff
@@ -50,7 +63,7 @@ public void shouldGenerateFiling() {
 
     @Test
     @DisplayName("Tests the unsuccessful creation of the ixbrl - filing is null")
-    void shouldNotGenerateFiling() throws IOException, NoSuchAlgorithmException {
+    void shouldNotGenerateFiling() {
```

Comment:
The unit tests are missing for the scenarios where the transaction and company account are null/not found

Reply:
43107d8